<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZGeometryClass</class>
 <widget class="QMainWindow" name="ZGeometryClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>850</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>ZGeometry</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QWidget" name="widgetOptions" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="layoutOptions">
       <item>
        <widget class="QComboBox" name="objSelectBox">
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>All</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox1"/>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox2"/>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layoutWorking">
      <item>
       <widget class="GLMeshWidget" name="glMeshWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>4</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>800</width>
          <height>600</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="splitterConsole">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>0</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QPlainTextEdit" name="consoleOutput">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>3</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
          </font>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="consoleInput">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1066</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionClone"/>
    <addaction name="actionDeformExperimental"/>
    <addaction name="actionFilterExperimental"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <widget class="QMenu" name="menuCurvature">
     <property name="title">
      <string>Curvature</string>
     </property>
     <addaction name="actionMeanCurvature"/>
     <addaction name="actionGaussCurvature"/>
    </widget>
    <widget class="QMenu" name="menuDisplay_Type">
     <property name="title">
      <string>Display mode</string>
     </property>
     <addaction name="actionDisplayPointCloud"/>
     <addaction name="actionDisplayWireframe"/>
     <addaction name="actionDisplayMesh"/>
    </widget>
    <addaction name="menuDisplay_Type"/>
    <addaction name="actionShowRefPoint"/>
    <addaction name="actionShowSignature"/>
    <addaction name="actionColorLegend"/>
    <addaction name="separator"/>
    <addaction name="menuCurvature"/>
    <addaction name="actionEigenfunction"/>
    <addaction name="actionHKS"/>
    <addaction name="actionSGWS"/>
    <addaction name="actionDiffPosition"/>
    <addaction name="separator"/>
    <addaction name="actionMexicanHatWavelet1"/>
    <addaction name="actionMexicanHatWavelet2"/>
    <addaction name="actionExperimental"/>
   </widget>
   <widget class="QMenu" name="menuCompute">
    <property name="title">
     <string>Compute</string>
    </property>
    <addaction name="actionComputeLaplacian"/>
    <addaction name="actionSGWSFeatures"/>
    <addaction name="actionSGW"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuCompute"/>
   <addaction name="menuDisplay"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionComputeLaplacian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Laplacian + MHB</string>
   </property>
  </action>
  <action name="actionEigenfunction">
   <property name="text">
    <string>Eigenfunction</string>
   </property>
  </action>
  <action name="actionMexicanHatWavelet1">
   <property name="text">
    <string>Mexican-hat Wavelet 1</string>
   </property>
  </action>
  <action name="actionMexicanHatWavelet2">
   <property name="text">
    <string>Mexican-hat Wavelet 2</string>
   </property>
  </action>
  <action name="actionExperimental">
   <property name="text">
    <string>Experimental</string>
   </property>
  </action>
  <action name="actionShowRefPoint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show reference point</string>
   </property>
  </action>
  <action name="actionMeanCurvature">
   <property name="text">
    <string>Mean Curvature</string>
   </property>
  </action>
  <action name="actionGaussCurvature">
   <property name="text">
    <string>Gauss Curvature</string>
   </property>
  </action>
  <action name="actionClone">
   <property name="text">
    <string>Clone</string>
   </property>
  </action>
  <action name="actionDeformExperimental">
   <property name="text">
    <string>Deform Experimental</string>
   </property>
  </action>
  <action name="actionDisplayPointCloud">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Point Cloud</string>
   </property>
  </action>
  <action name="actionDisplayWireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wireframe</string>
   </property>
  </action>
  <action name="actionDisplayMesh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mesh</string>
   </property>
  </action>
  <action name="actionShowSignature">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show signature</string>
   </property>
  </action>
  <action name="actionSGWSFeatures">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SGWS Features</string>
   </property>
  </action>
  <action name="actionHKS">
   <property name="text">
    <string>HKS</string>
   </property>
  </action>
  <action name="actionSGWS">
   <property name="text">
    <string>SGWS</string>
   </property>
  </action>
  <action name="actionFilterExperimental">
   <property name="text">
    <string>Filter Expreimental</string>
   </property>
  </action>
  <action name="actionSGW">
   <property name="text">
    <string>SGW</string>
   </property>
  </action>
  <action name="actionDiffPosition">
   <property name="text">
    <string>Diff Position</string>
   </property>
  </action>
  <action name="actionColorLegend">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show color legend</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLMeshWidget</class>
   <extends>QWidget</extends>
   <header>glmeshwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
